ci/pipelines/toolbox-deploy-ecr.yml

```diff
@@ -0,0 +1,102 @@
+resources:
+  - name: toolbox-deploy-ecr
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master 
+      paths:
+        - ci/pipelines/toolbox-deploy-ecr.yml
+  # Github Releases
+  - name: git-toolbox-staging-release
+    type: git
+    icon: github
+    source: &github-release-source
+      uri: https://github.com/alphagov/pay-toolbox
+      branch: master
+      tag_regex: "alpha_staging-2-(.*)"
+  # ECR registry resource
+  - name: ecr-registry-test
```

Comment:
Let's rename this resource `ecr-registry-staging`, as we're going to need both of them in this pipeline, right?